### PR TITLE
TECH-738 upgrade wagtail

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -65,7 +65,7 @@ unicodecsv==0.14.1
 virtualenv==15.0.3
 virtualenv-clone==0.2.6
 virtualenvwrapper==4.7.2
-wagtail==1.7
+wagtail>=1.8,<1.9
```
Comment (cjmabry) [inline requirements.txt]: What's the reason we don't just use `==1.8` - is it because we want the latest minor version?

Comment (ericbroder) [reply requirements.txt]: We want the latest patch version. The minor version is `1.8.x`, which won't change. But we want patch/bug fixes that are applied to the minor version we are on.

Comment (ericbroder) [reply requirements.txt]: see: http://docs.wagtail.io/en/v1.13.1/releases/upgrading.html#upgrade-process

**Comment by ericbroder** [comment]
dont merge until code freeze is over

File: requirements.txt (hunk 1 above)
Comment (cjmabry) [reply requirements.txt]: Cool, that's what I assumed, thanks!

**Comment by cjmabry** [comment]
Don't merge until after today's deploy.

**Pull request merged**